### DISCUSSION TITLE
How to get Scratch Addons Pre-Release

### DISCUSSION THREAD

**Discussion by Unkrown6YT** [ScratchAddons/ScratchAddons#5109] [Help] [answered]
Tell me, please help :<

**Comment by Samq64** [comment]
https://github.com/ScratchAddons/ScratchAddons/archive/refs/heads/master.zip

**Comment by retronbv** [comment, marked as answer]
https://github.com/ScratchAddons/ScratchAddons#from-source

**Reply by Unkrown6YT** [reply]
Thanks
